dm/dm/dm-ansible/scripts/DM-Monitor-Professional.json

```
@@ -3716,12 +3716,12 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "dm_relay_binlog_file{instance=~\"$instance\", node=\"relay\"} - ON(instance, task) dm_syncer_binlog_file{instance=~\"$instance\", task=~\"$task\", node=\"syncer\"}",
+          "expr": "dm_relay_binlog_file{instance=~\"$instance\", node=\"relay\"} - ON(instance) dm_syncer_binlog_file{instance=~\"$instance\", task=~\"$task\", node=\"syncer\"}",
           "format": "time_series",
           "hide": false,
           "interval": "$interval",
           "intervalFactor": 2,
-          "legendFormat": "{{task}} - {{instance}}",
+          "legendFormat": "{{{instance}}",
```

```suggestion
          "legendFormat": "{{instance}}",
```

could you provide an environment to let me verify the layout?

---

thanks! my bad , add preview link in about review comments now